src/uproot/models/RNTuple.py

```diff
@@ -266,9 +302,7 @@ def to_akform(self):
         recordlist = []
         topnames = self.keys()
         seen = []
-        for (i, fr) in enumerate(frs):
-            if fr.parent_field_id == i:
-                topnames.append(fr.field_name)
+        for (i, _) in enumerate(frs):
```

Comment:
Also unnecessary parentheses. Better style to let them hang free.

```suggestion
        for i, _ in enumerate(frs):
```